### DISCUSSION TITLE
Is it possible to "hot" start record RTMP stream

### DISCUSSION THREAD

**Discussion by sav1novhub** [bluenviron/mediamtx#2394] [Questions] [answered]
I want to start record from RTMP stream not from start, maybe from 1 minute at start record. Is it possible to control record through API?

**Comment by aler9** [comment, marked as answer]
Hello, it's possible to control recording through the API, but at the moment toggling recording causes the stream to restart. In order to solve the issue, i've created a feature request here: #2395

**Comment by aler9** [comment]
This has been fixed by #2434

**Reply by AshishRaj3320** [reply]
please provide the api endmpoint for record , if any RTMP or RTSP stream is running.

**Reply by AshishRaj3320** [reply]
is it possible to record manually from API?
